themes/phenomic-theme-base/src/layouts/Page/index.js

```diff
@@ -30,22 +30,24 @@ const Page = (
 
   const metaTitle = head.metaTitle ? head.metaTitle : head.title
 
+  const externalUrlCheck = new RegExp('^https?://');
+  const socialImage = externalUrlCheck.test(head.hero) ?
+  head.hero : joinUri(process.env.PHENOMIC_USER_URL, head.hero)
+
```

Comment:
I think we  should move this into the loader part, processing this on every render isn't good for performance wise